packages/ember-flight-icons/src/components/flight-icon.ts

```diff
@@ -6,17 +6,49 @@
 import Component from '@glimmer/component';
 import { guidFor } from '@ember/object/internals';
 import { assert } from '@ember/debug';
-
 import { iconNames } from '@hashicorp/flight-icons/svg';
 
-export default class FlightIconComponent extends Component {
-  constructor() {
-    super(...arguments);
+import type { IconName } from '@hashicorp/flight-icons/svg';
+
+export interface FlightIconSignature {
+  Args: {
+    /**
+     * The color of the icon. Defaults to `currentColor`.
+     */
```

Comment:
Currently we don't have inline comments to describe the arguments in the other signatures. Do you mind removing them? We may decide later to do it systematically across all the codebase, but for now I prefer to have consistency in how we declare the signatures, to avoid having a jungle of formats. Thank you